_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -127,11 +129,41 @@ This `combobox` has the required properties `aria-controls` and `aria-expanded`.
 </ul>
 ```
 
+#### Passed Example 7
+
+This `checkbox` has its required `aria-checked` property [implicitly set][aria set implicit].
+
+```html
+<label>
+	<input type="checkbox" />
+	Check me
+</label>
+```
+
+#### Passed Example 8
+
+This `menuitemcheckbox` has its required `aria-checked` property [implicitly set][aria set implicit].
```

Comment:
```suggestion
This `menuitemcheckbox` has its required `aria-checked` property [implicitly set][aria set implicit] because type="checkbox" is set on the element.
```

If I am correctly understanding, `aria-checked` is only set because of `type="checkbox"`? Should this be clarified and then a failed example created (if we don't have one already)?

This `menuitemcheckbox` does not have its required `aria-checked` property [implicitly set][aria set implicit]. 
```html
<label>
	<input role="menuitemcheckbox" />
	Check me
</label>
```